app/src/main/java/org/mozilla/rocket/banner/BannerAdapter.java

```diff
@@ -62,18 +78,19 @@ public BannerViewHolder onCreateViewHolder(@NonNull ViewGroup parent, @ViewType
             case FourSitesViewHolder.VIEW_TYPE:
                 return new FourSitesViewHolder(parent, onClickListener);
             case BasicViewHolder.VIEW_TYPE:
+            case UNKNOWN_VIEW_TYPE:
```

Comment:
Since we already filter unknown type from input parsing, should we also throw an exception on non-release build? to raise the problem?